src/pages/plugins/tutorials/plugin-lifecycle-hooks/index.md

```diff
@@ -0,0 +1,29 @@
+# Plugin Lifecycle events
+You can detect whether or not a plugin panel is visible with the help of UXP APIs
+
+<InlineAlert variant="info" slots="text"/>
+
+This is only applicable to plugins with a panel. You cannot detect the execution of a command plugin via these hooks. //TODO verify fact about command plugins and hooks. Also, for plugins that have multiple panels, you cannot differentiate between them, at present. Familiarize yourself with [plugin entrypoints](../../concepts/entry-points/) to know the difference between a command and panel plugin.
```

Comment:
@kareemmyk  Please verify this statement: "This is only applicable to plugins with a panel. You cannot detect the execution of a command plugin via these hooks."

Reply:
@anoopvalomkot  @kareemmyk  Not able to make these events work. Can you please check?

Reply:
@anoopvalomkot , UXP team will share the information soon to @padmkris123

Reply:
@kareemmyk I tried this, don't see these events on closing/opening the panel

Reply:
@padmkris123 
As discussed, please refer [3P Plugin Lifecycle Alignmen](https://wiki.corp.adobe.com/pages/viewpage.action?spaceKey=UXP&title=3P+Plugin+Lifecycle+Alignment+-+Design+Spec)

cc: @anoopvalomkot